idd/Energy+.idd.in

```diff
@@ -49334,7 +49334,7 @@ Coil:Cooling:DX:CurveFit:Speed,
        \note Rated sensible heat ratio (gross sensible capacity/gross total capacity)
        \note sensible and total capacities do not include supply fan heat
        \type real
-       \minimum 0.5
+       \minimum 0.1
```

Comment:
@lgu @Myoldmopar There's still one small IDD change here.

Reply:
@Myoldmopar The change is necessary. Please update it.

Reply:
@lgu1234 This is now in develop.